ops/ccs-ops-misc/load_test/common/test_setup.py

```diff
@@ -27,6 +29,18 @@ def load_server_public_key() -> str:
     except KeyError:
         return False
 
+def load_stats_storage_config() -> StatsStorageConfig:
+    """Load the storage configuration for storing aggregated statistics.
+
+    Returns:
+        StatsStorageConfig: A dataclass representing the storage configuration for aggregated statistics
+    """
+
+    # TODO: This may not be the best spot for this method, since it's used during teardown not setup.
+    # Maybe add a new test_teardown.py, or move the loading of values from config to config.py?
```

Comment:
Yeah, I was thinking that as I was writing it. Tbh, I feel like _all_ of the loading functions in this file should all be moved to `config.py`, not just this single function.

I think I'll go ahead with that refactor and move everything that loads from the config into `config.py` so that `config.py` is all the config-related stuff and `test_setup.py` is just meant for code that sets up the Locust environment and related things.